crmsh/utils.py

```diff
@@ -3046,4 +3046,12 @@ def get_systemd_timeout_start_in_sec(time_res):
     res_min = re.search("(\d+)min", time_res)
     start_timeout += 60 * int(res_min.group(1)) if res_min else 0
     return start_timeout
+
+
+def any_unclean_nodes():
+    """
+    Get the UNCLEAN node list if any
+    """
+    out = get_stdout_or_raise_error("crm_mon -1")
```

Comment:
I wonder  it might be easy to convert the follow to pythonic crmsh code. It more elegant than grep the human text.
`crm_mon --output-as=xml | xmllint --xpath '//nodes/node[@unclean="true"]' - `

Reply:
It deserves a dedicate fix in general. Resolve this by,
https://github.com/ClusterLabs/crmsh/issues/902